packages/mdc-textfield/helper-text/_mixins.scss

```diff
@@ -37,13 +37,13 @@
 // Private mixins
 
 @mixin mdc-text-field-helper-text-color_($color) {
-  + .mdc-text-field-helper-text {
+  + .mdc-text-field-helper-line .mdc-text-field-helper-text {
```

Comment:
Yes, since `helper-line` is the new immediate sibling of `mdc-textfield` changed it to target it using sibling selector `+` with child element `helper-text` selector.